__tests__/app.test.js

```diff
@@ -285,66 +269,52 @@ describe("6. GET /api/reviews (queries)", () => {
         expect(reviews).toBeInstanceOf(Array);
         expect(reviews).toHaveLength(13);
         expect(reviews).toBeSortedBy("title", { ascending: true });
-        reviews.forEach((reviews) => {
-          expect(reviews).toEqual(
-            expect.objectContaining({
-              owner: expect.any(String),
-              title: expect.any(String),
-              review_id: expect.any(Number),
-              category: expect.any(String),
-              review_img_url: expect.any(String),
-              created_at: expect.any(String),
-              votes: expect.any(Number),
-              designer: expect.any(String),
-              comment_count: expect.any(Number),
-            })
-          );
-        });
       });
   });
 
-  test("status:200, category and sorts by asc)", () => {
-    return request(app)
-      .get("/api/reviews?category=social deduction&sort_by=title&order=ASC")
-      .expect(200)
-      .then(({ body }) => {
-        const { reviews } = body;
-        expect(reviews).toBeInstanceOf(Array);
-        expect(reviews).toHaveLength(11);
-        expect(reviews).toBeSortedBy("title", { ascending: true });
-        reviews.forEach((reviews) => {
-          expect(reviews).toEqual(
-            expect.objectContaining({
-              owner: expect.any(String),
-              title: expect.any(String),
-              review_id: expect.any(Number),
-              category: expect.any(String),
-              review_img_url: expect.any(String),
-              created_at: expect.any(String),
-              votes: expect.any(Number),
-              designer: expect.any(String),
-              comment_count: expect.any(Number),
-            })
-          );
-        });
+test("status:200, category and sorts by asc)", () => {
+  return request(app)
+    .get("/api/reviews?category=social deduction&sort_by=title&order=ASC")
+    .expect(200)
+    .then(({ body }) => {
+      const { reviews } = body;
+      expect(reviews).toBeInstanceOf(Array);
```

Comment:
The assertion of this being an array does not ned to be made as you have a forEach which is an array method itself, so would make that check for you.